### PR TITLE
Added new command "config"

### LINKED ISSUE #10: Add `PatchConfig` command to `inf`
Child task of infinimesh/infinimesh#661

### PR DESCRIPTION
**Pull request by dimquaa**
Resolves #10

### CODE REVIEW

**Review by gorobot-nz — APPROVED** [review]

File: cmd/devices.go [hunk 1]
```diff
@@ -152,6 +153,51 @@ var toggleDeviceCmd = &cobra.Command{
 	},
 }
 
+var patchConfigDeviceCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Patch config infinimesh device",
+	RunE: func(cmd *cobra.Command, args []string) error {
```
Comment (slntopp) [inline cmd/devices.go:159]: Add arts length validation

File: cmd/devices.go [hunk 2]
```diff
@@ -152,6 +153,51 @@ var toggleDeviceCmd = &cobra.Command{
 	},
 }
 
+var patchConfigDeviceCmd = &cobra.Command{
+	Use:   "config",
```
Comment (slntopp) [inline cmd/devices.go:157]: Document the arguments in "use" section
See other commands

**Review by slntopp — CHANGES_REQUESTED** [review]

File: cmd/devices.go (hunk 1 above)
Comment (dimquaa) [reply cmd/devices.go:159]: do you mean args ?

Comment (dimquaa) [reply cmd/devices.go:159]: check length of the incoming uuid?

Comment (slntopp) [reply cmd/devices.go:159]: There is a Args field in `cobra.Command` where you can supply cobra validation func, which would check the length. So you don't need to do `if len(...) {...}` youself

**Review by slntopp — APPROVED** [review]

**Pull request merged**